MyConfig.h

```diff
@@ -270,7 +270,7 @@
  */
 // legacy
 #ifdef MY_RADIO_NRF24
-//MY_RADIO_NRF24 is deprecated
+#warning MY_RADIO_NRF24 is deprecated, use MY_RADIO_RF24 instead.
```

Comment:
In addition to a compiler warning when defined, I think it might also be a good idea to add a doxygen block marking the define @deprecated (see https://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmddeprecated)

Something along the lines of
```
 /**
  * @def MY_RADIO_NRF24
  * @brief Define this to use a RF24-based radio transport for sensor network communication.
  * @deprecated This flag is deprecated and replaced by @ref MY_RADIO_RF24
  */
 //#define MY_RADIO_RF24
```

To allow doxygen to pick up the define, it also need to be defined below under the doxygen specific segment (see https://github.com/mysensors/MySensors/pull/1159/commits/a7299d9d925213f8a7c029fc47e255514c876488#diff-cf14b6301beb3a4dbaded9b95bb190feL2295)

Reply:
Hmm, okay, but where exactly do I add it?

Reply:
At my comment

Reply:
If you don't have doxygen locally, you can examine the results from the build server by clicking "Details" next to the "Toll gate (Documentation)" check below.

Reply:
@Avamander is there anything we can do to assist you?

Reply:
@Avamander Please submit this PR against __mysensors:development__, not mysensors:development-3.0.0